dev/hal_k8s_run.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The purpose of this script is to install Halyard at the latest released
+# version (output from our build process, not a Github release) and use
+# Halyard to deploy Spinnaker to Kubernetes at the versions specified in
+# the latest nightly version. After Spinnaker is instantiated on Kubernetes,
+# a Citest suite will run against it. This is outside the scope of this script.
+
+# It is assumed that this script will run in a container with the proper
+# environment variables and files present.
+
+set -e
+export TERMINFO=/usr/lib/terminfo
+export TERM=xterm
+
+mkdir /supporting_data
+echo $BUILD_KEY >> /supporting_data/build.json
+echo $DOCKER_KEY >> /supporting_data/docker.json
+
+curl -O https://storage.googleapis.com/kubernetes-release/release/v1.5.4/bin/linux/amd64/kubectl
+chmod +x kubectl
+mv kubectl /usr/local/bin/kubectl
+
+echo "Downloading Halyard..."
+wget https://raw.githubusercontent.com/spinnaker/halyard/master/InstallHalyard.sh
+
+# We need to make changes to the Halyard install script so it can run in a container.
+sed "s/^printf/echo/" -i InstallHalyard.sh
+sed "s/^start halyard/exit 0/" -i InstallHalyard.sh
+
+sudo bash InstallHalyard.sh -y
+echo "Configuring Halyard..."
+cat > /opt/spinnaker/config/halyard.yml  <<EOF
```

Comment:
Are you publishing BOMs? if not, the default halyard.yml that comes with your installation should be enough